lib/src/sass/_modals.scss

```diff
@@ -0,0 +1,59 @@
+@import './colors';
+@import './mixins';
+
+$modal-padding: 20px;
+$header-footer-height: 75px;
+$modal-border: 1px solid $gray-200;
+$body-height: 100%;
+
+.hc-modal-header {
+    padding: $modal-padding;
+    height: $header-footer-height;
+    border-bottom: $modal-border;
+    font-size: calculateRem(18px);
+    color: $offblack;
```

Comment:
I'd suggest using the `$text` value for color instead (it points to `$offblack`) - here and on any other styles.  Just in case we ever want to change our text color, it'll be easier if we just have to change it in one place.

Reply:
Done